sdk/search/azure-search-documents/azure/search/documents/_search_client.py

```diff
@@ -259,11 +270,21 @@ def search(self, search_text, **kwargs):
         search_fields_str = ",".join(search_fields) if search_fields else None
         search_mode = kwargs.pop("search_mode", None)
         query_language = kwargs.pop("query_language", None)
-        speller = kwargs.pop("speller", None)
-        answers = kwargs.pop("answers", None)
+        query_speller = kwargs.pop("query_speller", None)
         select = kwargs.pop("select", None)
         skip = kwargs.pop("skip", None)
         top = kwargs.pop("top", None)
+
+        query_answer = kwargs.pop("query_answer", None)
+        query_answer_count = kwargs.pop("query_answer_count", None)
+        answers = query_answer if not query_answer_count else '{}|count-{}'.format(query_answer, query_answer_count)
```

Comment:
Is it valid with query_answer = "None" and query_answer_count = 3?

Reply:
No, you will get a service error InvalidRequestParameters.

Reply:
Then should we add into the docstring it only works when query_answer is "extractive"?

Reply:
(I see you have it for query_caption_highlight)

Reply:
Fixed.